src/app/modules/ix-tables/services/empty.service.ts

```diff
@@ -23,6 +25,12 @@ export class EmptyService {
     title: this.translate.instant('Can not retrieve response'),
   };
 
+  private readonly emptySearchResultsConfig: EmptyConfig = {
+    title: this.translate.instant('No such items'),
```

Comment:
`No such items` will get extracted into translation files and people will translate it, though it is not used.